src/ce/real.hpp

```
@@ -0,0 +1 @@
+#include <ti/real>
```

What is the purpose of this file?

---

`<ti/real>` used to be `<real.hpp>`. Perhaps it should throw an error telling users to use the new name instead?

---

It used to be **ti**real.hpp, didn't it?

---

Yes, but I already fixed that.